pipelines/docker-build-rhtap/docker-build-rhtap.yaml

```diff
@@ -8,30 +8,7 @@ metadata:
     pipelines.openshift.io/used-by: build-cloud
   name: docker-build-rhtap
 spec:
-  finally:
-  - name: show-sbom
-    params:
-    - name: IMAGE_URL
-      value: $(tasks.build-container.results.IMAGE_URL)
-    taskRef:
-      name: show-sbom-rhdh
-      version: "0.1"
```

Comment:
It looks like the RHTAP pipelines also lost the show-sbom task, was that intended?